src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -369,15 +374,16 @@ module TacoDependencyInstaller {
                 logger.log(resources.getString("LicenseAgreement"));
             }
 
+            // If we accept prompts automatically, then return immediately
+            if (TacoGlobalConfig.acceptPrompts) {
+                return Q.resolve(true);
```

Comment:
So, decision is to not print such messages (as user should be expecting that and went out of his way to skip prompts), but I should have added a help entry to explain this new parameter, so I'll do that now